Classes/Categories/ITDatabaseOperationsQueue+Init.h

```diff
@@ -15,3 +15,4 @@
 - (instancetype)initWithModel:(NSManagedObjectModel *)model storeName:(NSString *)storeName storeType:(NSString *)storeType;
 
 @end
+
```

Comment:
line 14 - method signature change to "+ (instancetype)newOperationQueueWithModel..." as it returns non-autoreleased object